src/AvailableCommandsPacket.php

```diff
@@ -145,9 +145,12 @@ public static function convertArg(int $protocolId, int $type) : int{
 		}
 
 		return match ($type) {
+			self::ARG_TYPE_TARGET => 0x07,
 			self::ARG_TYPE_WILDCARD_TARGET => 0x08,
+			self::ARG_TYPE_FILEPATH => 0x10,
 			self::ARG_TYPE_STRING => 0x20,
-			self::ARG_TYPE_POSITION => 0x28,
+			self::ARG_TYPE_INT_POSITION => 0x28,
```

Comment:
If I'm not mistaken ARG_TYPE_INT_POSITION didn't exist before 1.18.30

Reply:
Isn't `ARG_TYPE_INT_POSITION` just a block position?

Reply:
Well only one of them existed pre 1.18.30. I never worked with any of them, but I doubt they changed the name of a thing that already existed & added a new implementation with the old name

Reply:
Just tested `0x28` in protocol `475` and it definitely exists

Reply:
0x28, yes but 0x29 didn't